hw-latest/fpga/setup_fpga.sh

```diff
@@ -5,7 +5,7 @@
 
 set -e
 
-CALIPTRA_ROOT=$(pwd)
+CALIPTRA_ROOT=$(realpath $(dirname "$( readlink -f -- "$0"; )")/../../)
```

Comment:
To prevent potential word splitting (SC2046) this should be quoted.

```suggestion
CALIPTRA_ROOT=$(realpath "$(dirname "$( readlink -f -- "$0"; )")"/../../)
```